catalog/task/sign-index-image/0.1/README.md

```diff
@@ -0,0 +1,15 @@
+# sign-index-image
+
+Creates an InternalRequest to sign an index image
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| requestJsonResults | The JSON result of the IIB build internal request | Yes | "{}" |
+| targetIndex | targetIndex signing image | Yes | - |
+| requestUpdateTimeout | Max seconds waiting for the status update| Yes | 360 |
+| signingPipelineImage | An image with CLI tools needed for the signing | Yes | quay.io/redhat-isv/operator-pipelines-images:released |
+| requester | Name of the user that requested the signing, for auditing purposes | No | - |
+| configMapName |ConfigMap Name for this request | No | hacbs-signing-pipeline-config |
```

Comment:
there should be one space before ConfigMap, not 0 or 2 :slightly_smiling_face:

Reply:
fixed!